src/api/feedback.api.js

```diff
@@ -50,12 +39,14 @@ export default async function sendFeedback(subject, text, options) {
             kmlData = kml
         }
 
+        const permalink = sanitizeUrl(window.location.href)
+
```

Comment:
I think helpdesk might want to know if the user had some crosshair or geolocation, so no need to "sanitize" it (they want to know the state of the viewer for the feedback they received)

Reply:
@pakb ok, I remove the last commit so it doesn't contain sanitazion.